tests/acceptance/features/webUIFiles/hiddenFile.feature

```diff
@@ -11,6 +11,7 @@ Feature: Hide file/folders
 
   @smokeTest
   @issue-900
+  @skipOnIphoneResolution @issue-1270
```

Comment:
#1270 was fixed meanwhile

Reply:
#1270 is about the create `+ NEW` button